docs/flatcar-linux/packet.md

```diff
@@ -307,20 +292,35 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 
 #### DNS Zone
 
-Clusters create a DNS A record `${cluster_name}.${dns_zone}` to resolve a network load balancer backed by controller instances. This FQDN is used by workers and `kubectl` to access the apiserver(s). In this example, the cluster's apiserver would be accessible at `tempest.aws.example.com`.
+Clusters create few DNS A records to resolve to controller instances. For instance `${cluster_name}.${dns_zone}` is used by workers and `kubectl` to access the apiserver(s). In this example, the cluster's apiserver would be accessible at `tempest.myclusters.example.com`.
```

Comment:
```suggestion
Clusters create few DNS A records to resolve to controller instances. For example `${cluster_name}.${dns_zone}` is used by workers and `kubectl` to access the apiserver(s). In this example, the cluster's apiserver would be accessible at `tempest.myclusters.example.com`.
```

`instance` => `example` to avoid using word `instance(s)` twice.